test/ValidatorSet/Inspector.test.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable node/no-extraneous-import */
+import { loadFixture, time } from "@nomicfoundation/hardhat-network-helpers";
+import * as hre from "hardhat";
+
+import { expect } from "chai";
+import { DENOMINATOR, VALIDATOR_STATUS, VESTING_DURATION_WEEKS, WEEK } from "../constants";
+import { registerValidator } from "../helper";
+
+export function RunInspectorTests(): void {
+  describe("Ban a validator", function () {
+    it("should revert if try to ban zero address", async function () {
```

Comment:
Why banning the zero address is a problem in case it has some balance? I don't think we need this check in practise.

Reply:
Well, I just thought there is no way the zero address to become a validator, but yeah, even without this check, the functionality will work fine.